src/main/resources/application.yaml

```diff
@@ -64,6 +64,9 @@ oneops:
     svc-user: ${KEYWHIZ_USER}
     svc-password: ${KEYWHIZ_PASS}
     secret-max-size: ${KEYWHIZ_SECRET_MAX_SIZE:350000}
+    connect-timeout: ${KEYWHIZ_CONNECT_TIMEOUT}
```

Comment:
Add the default value here like in max size.

Reply:
ok

Reply:
${KEYWHIZ_CONNECT_TIMEOUT:5}
Like this: ^

Reply:
yes, but the default value should be 10 seconds.